### PR TITLE
Improved macos guide

### PR DESCRIPTION
**Pull request by defnax**

### CODE REVIEW

File: build_scripts/OSX/MacOS_X_InstallGuide.md [hunk 1]
```diff
@@ -108,7 +110,7 @@ Edit RetroShare.pro
 
     CONFIG += c++14 rs_macos11.1
 
-and then retroshare.pri
+Edit retroshare.pri
```
Comment (csoler) [inline build_scripts/OSX/MacOS_X_InstallGuide.md:114]: it's surprising that the user would need to edit retroshare.pri. Cannot this be done in a specific MacOS section?

Comment (defnax) [reply build_scripts/OSX/MacOS_X_InstallGuide.md:114]: not possible witthout edit, its something hardcoded gio developed that thing 
new pr her
https://github.com/RetroShare/RetroShare/pull/2797

Comment (defnax) [reply build_scripts/OSX/MacOS_X_InstallGuide.md:114]: i had lots of issues in the passt we need edit always macos section to find sdk, to give via parameter not working

Comment (defnax) [reply build_scripts/OSX/MacOS_X_InstallGuide.md:114]: look retifs howto he said too for macosx find sdk is hardcoded 
https://decovar.dev/blog/2021/04/01/macos-building-retroshare/

**Pull request closed without merging**